python-backend/app/api/mines/mine/resources/mine.py

```diff
@@ -122,6 +120,7 @@ def post(self, mine_no_or_guid=None):
                 mine_name=data['name'],
                 mine_note=note if note else '',
                 major_mine_ind=major_mine_ind,
+                mine_region=region,
```

Comment:
For consistency, should we rename the `region` key to `mine_region`? Like how `major` was renamed to `major_mine_ind`